src/patternfly/components/List/docs/code.md

```diff
@@ -1,11 +1,10 @@
 ## Overview
 
-Typically used with a modifier class.
+Non-inline lists can be nested up to any level.
 
 ## Usage
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.pf-c-list` | `<ul>` |  Initiates a list with the same styling as lists in the content component. **Required** |
-| `.pf-m-inline` | `.pf-c-list` |  Modifies for inline list style. |
-| `.pf-m-grid` | `.pf-c-list` |  Modifies for grid list style. |
+| `.pf-c-list` | `<ul>, <ol>` | Initiates a list. Must be used with a modifier. |
```

Comment:
Must be used with a modifier, but the only one we have is `.pf-m-inline` - looks like it actually does not need to be used with a modifier?

Reply:
yep! updated.